### PR TITLE
Sx17 turn all plugs off

### PR DESCRIPTION
**Pull request by daliamaarek**
#59

@Youmna-Salah: code
@AmirMaurice : UML
@maggie-moheb : test
@ahmad3m3 : scenario
@azamzamy : documenation

kz345

### CODE REVIEW

**Comment by maggie-moheb** [comment]
tests are great :+1:

**Comment by AmirMaurice** [comment]
@daliamaarek Kindly add the response back to the user in the sequence diagram

**Comment by daliamaarek** [comment]
@AmirMaurice There is no response :angel:

File: SmartX/app/src/main/java/com/smartx/cookies/smartx/myAPI.java [hunk 1]
```diff
@@ -163,5 +163,14 @@ void getNotes(@Path("user_id") String userid, @Path("room_id") String roomid, @P
     @FormUrlEncoded
     @PUT("/users/{userID}/rooms/{roomID}/devices/{deviceID}")
     void addToFavorites(@Path("userID") String userID, @Path("roomID") String roomID, @Path("deviceID") String deviceID, @Field("device[favorite]") String favorite, Callback<Device> callback);
+
```
Comment (azamzamy) [inline SmartX/app/src/main/java/com/smartx/cookies/smartx/myAPI.java:183]: Could you document your api methods, please?

**Comment by daliamaarek** [comment]
@Youmna-Salah @azamzamy done!

**Comment by AmirMaurice** [comment]
@daliamaarek ok :+1:

**Pull request merged**